ui-v2/app/templates/dc/services/show/services.hbs

```diff
@@ -7,7 +7,7 @@
         <a href={{env 'CONSUL_TERMINATING_GATEWAYS_URL'}} target="_blank">step-by-step guide.</a>
         
       </p>
-      <ListCollection @cellHeight={{73}} @items={{gateway.Services}} class="gateway-services-list" as |item index|>
+      <ListCollection @cellHeight={{73}} @items={{gateway.Services}} class="consul-gateway-services-list" as |item index|>
```

Comment:
Note for the future, feels like this `73` could be added in the component itself, at least as the default?